src/libs/HttpUtils.js

```diff
@@ -5,15 +5,15 @@ import CONFIG from '../CONFIG';
 import CONST from '../CONST';
 import ONYXKEYS from '../ONYXKEYS';
 import HttpsError from './Errors/HttpsError';
-import shouldUseStagingServer from './shouldUseStagingServer';
 import getPlatform from './getPlatform';
+import proxyConfig from '../../config/proxyConfig';
 
 // Desktop and web use staging config too so we we should default to staging API endpoint if on those platforms
 const shouldDefaultToStaging = _.contains([CONST.PLATFORM.WEB, CONST.PLATFORM.DESKTOP], getPlatform());
```

Comment:
This seems to be the reason you're hitting staging @mountiny 

The default should always be false now